docs/fastdds/getting_started/simple_python_app/includes/run.rst

```diff
@@ -0,0 +1,17 @@
+Putting all together
+^^^^^^^^^^^^^^^^^^^^
+
+Finally, from the build directory, run the publisher and subscriber applications from two terminals.
+
+.. code-block:: bash
+
+    python3 HelloWorldPublisher.py
+    python3 HelloWorldSubscriber.py
```

Comment:
This does not work on a new terminal with a clean environment. Maybe add a line or something regarding sourcing the Python Bindings installation

Reply:
In section [dependencies](https://github.com/eProsima/Fast-DDS-docs/blob/6c1422b84d31c06f96765ec194f67003608e592a/docs/fastdds/getting_started/simple_app/includes/dependencies.rst#L3) it is explained how source the installation. It is the same as in already existing C++ "getting started" example.